src/Microsoft.AspNetCore.Blazor.Razor.Extensions/BlazorExtensionInitializer.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.Language.Extensions;
+
+namespace Microsoft.AspNetCore.Blazor.Razor
+{
+    public class BlazorExtensionInitializer : RazorExtensionInitializer
```

Comment:
This is the recipe for creating a razor 'plugin'.

I grouped a bunch of the old configuration code here for now for ease of maintenance.